.yamato/hdrp-osx-metal.yml

```diff
@@ -30,8 +30,8 @@ HDRP_OSX_Metal_playmode_trunk:
             paths:
               -  "**/test-results/**"
               -  "TestProjects/HDRP_Tests/Logs/*.log"
-HDRP_OSX_Metal_playmode_XR_trunk:
```

Comment:
similar to before, why are we excluding xr on metal?

Reply:
oh i thought this was a change that happened in trunk 🤔

Reply:
oh i guess not, i will undo this